package.json

```diff
@@ -41,6 +43,7 @@
     "husky": "^0.13.2",
     "istanbul": "^0.4.5",
     "mocha": "^3.2.0",
-    "nodemon": "^1.11.0"
+    "nodemon": "^1.11.0",
+    "snyk": "https://registry.npmjs.org/snyk/-/snyk-1.25.2.tgz"
```

Comment:
Looks like it is a bug https://github.com/npm/npm/issues/15050
I've updated to just the version number and re-generated the shrinkwrap